components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -52,11 +64,28 @@ class TextFieldTests: XCTestCase {
       XCTAssertEqual(textField.font, textFieldCopy.font)
       XCTAssertEqual(textField.hidesPlaceholderOnInput, textFieldCopy.hidesPlaceholderOnInput)
       XCTAssertEqual(textField.isEnabled, textFieldCopy.isEnabled)
+
+      if let leadingViewCopy = textField.leadingView as? UILabel {
```

Comment:
This test makes sure the copy's propertys match the original. The copy happens on the line 
```    if let textFieldCopy = textField.copy() as? MDCTextField {```